Travel Scheduler/View Controllers/MoreOptionViewController.m

```diff
@@ -46,8 +47,7 @@ - (nonnull __kindof UICollectionViewCell *)collectionView:(nonnull UICollectionV
 
 - (NSInteger)collectionView:(nonnull UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section
 {
-    //return self.places.count;
-    return 20; //TESTING
+    return 20;
```

Comment:
Should return the total count in the array.

Reply:
done!

Reply:
Shouldn't this then be `self.places.count`?

Reply:
Yes! It is like this now